server/config/constants.js

```diff
@@ -0,0 +1,10 @@
+const ROLE_ENUM = {
+  0: 'SUPER_ADMIN',
```

Comment:
why redundancy here?

Reply:
If we want to get what is 0 or 1 or 2 then it will directly give the value. yes, because of this there is redundancy but it will give value in O(n)